tests/check_medical_allowed_amount.sql

```diff
@@ -0,0 +1,34 @@
+/* This is taken from the claims mapping guide, which says, 
+"The expectation is that the sum of paid_amount, coinsurance_amount, copayment_amount, and deductible_amount will be equivalent to allowed_amount."
+*/
+{{ config(
+     enabled = var('claims_preprocessing_enabled',var('claims_enabled',var('tuva_marts_enabled',False)))
+ | as_bool,
+     tags = ['dqi', 'tuva_dqi_sev_2', 'dqi_service_categories', 'dqi_ccsr', 'dqi_cms_chronic_conditions',
```

Comment:
```suggestion
     tags = ['dqi', 'tuva_dqi_sev_4', 'dqi_service_categories', 'dqi_ccsr', 'dqi_cms_chronic_conditions',
```
@amartinson193 Any reason why this is sev 2 versus sev 4